docker-compose.yml

```diff
@@ -13,21 +13,27 @@ services:
       - "5432"
   app:
     <<: *defaults
-    build: .
+    build:
+      context: .
+      dockerfile: ./compose/development/app/Dockerfile
     environment:
       CANDIS_DATABASE_HOST: db
       CANDIS_CACHE_HOST: cache
     networks:
       - db
       - cache
       - proxy
-
+    
     depends_on:
       - db
     volumes:
-      - .:/app
```

Comment:
this was correct.

Reply:
We don't need to mount the files other than these two. Because in development we would be changing mostly the /candis directory only